client/executor/wasmtime/src/host.rs

```diff
@@ -46,7 +46,7 @@ unsafe impl Send for SandboxStore {}
 /// many different host calls that must share state.
 pub struct HostState {
 	sandbox_store: SandboxStore,
-	allocator: FreeingBumpHeapAllocator,
+	allocator: Option<FreeingBumpHeapAllocator>,
```

Comment:
Nit: does it make sense to briefly mention the rationale for using the `Option` here?